scipy/signal/windows/windows.py

```diff
@@ -2030,6 +2030,7 @@ def _fftautocorr(x):
     ('taylor', 'taylorwin'): (taylor, False),
     ('triangle', 'triang', 'tri'): (triang, False),
     ('tukey', 'tuk'): (tukey, True),
+    ('dpss'): (dpss, True),
```

Comment:
The left is not a tuple but a plain string, so we should do instead:
```suggestion
    ('dpss',): (dpss, True),
```
Otherwise in the iteration below we'll end up with `'d'`, `'p'`, and `'s'` strings all pointing to dpss as well.

Also it would be good to keep it alphabetized, so it should go after `cosine`